diskv.go

```diff
@@ -403,6 +433,9 @@ func (d *Diskv) EraseAll() error {
 	defer d.mu.Unlock()
 	d.cache = make(map[string][]byte)
 	d.cacheSize = 0
+	if d.TempDir != "" {
+		os.RemoveAll(d.TempDir) // errors ignored
```

Comment:
oops, nevermind... I missed that this was in EraseAll()... I thought it was a cleanup case in writeStreamWithLock